rasa/core/brokers/kafka.py

```diff
@@ -45,6 +46,10 @@ def __init__(
                 sender_id or not
             sasl_username: Username for plain authentication.
             sasl_password: Password for plain authentication.
+            sasl_mechanism: Authentication mechanism when security_protocol is
+                configured for SASL_PLAINTEXT or SASL_SSL.
+                Valid values are: PLAIN, GSSAPI, OAUTHBEARER, SCRAM-SHA-256,
```

Comment:
How about validating that the values are in this range? Or is this done by the library? If it's done by the library and it raises an exception this should wrapped in an exception type which inherits from `RasaException` so that we don't see this event in sentry

Reply:
Ah yeah it's handled by the library and raises an AssertionError